js/utils/templateHelpers.js

```diff
@@ -1,7 +1,22 @@
 import app from '../app';
+import { getHiRez } from './responsive';
+import { getLargeWidth } from './responsive';
+import { getSmallHeight } from './responsive';
 
 export function polyT(...args) {
   return app.polyglot.t(...args);
 }
 
 export const getServerUrl = app.getServerUrl;
+
+export function hiRez() {
```

Comment:
This can be written more concisely as `export { getHiRez as hiRez };`... but why the name change? Without the name change, it could just be `export { getHiRez };`.